roxctl/common/flags/endpoint.go

```diff
@@ -79,6 +81,9 @@ func AddConnectionFlags(c *cobra.Command) {
 func EndpointAndPlaintextSetting() (string, bool, error) {
 	endpoint = flagOrSettingValue(endpoint, *endpointChanged, env.EndpointEnv)
 	if !strings.Contains(endpoint, "://") {
+		if _, _, err := net.SplitHostPort(endpoint); err != nil {
+			return "", false, errox.InvalidArgs.Newf("invalid endpoint: %s, the scheme should be: http(s)://<endpoint>:<port>", err.Error())
```

Comment:
```suggestion
			return "", false, errox.InvalidArgs.CausedBy(err)
```
This will be more in line with at least another error down the file and will render like "invalid arguments: missing port" which should be exactly the right amout of detail at this level.